articles/rules/index.md

```diff
@@ -13,7 +13,7 @@ An App initiates an authentication request to Auth0 (__Step 1__), Auth0 routes t
 Here are a few examples. You could:
 
 * Bring information from your own databases and add it to the user profile object.
-* Create authorization rules based on complex logic (anything that can be written with node.js).
+* Create authorization rules based on complex logic (anything that can be written with JavaScript).
```

Comment:
I took this discussion into account https://github.com/auth0/rules/issues/17 it seem to be the conclussion they arrived to (considering @jfromaniello comment "I think there is an issue with that part of the documentation, because it is not node."), but if it's better as it was I can close this PR. Let me know